gradle/test-libs.versions.toml

```diff
@@ -19,3 +20,5 @@ testcontainers-core = {module = "org.testcontainers:testcontainers", version.ref
 junit-system-rules = {module = "com.github.stefanbirkner:system-rules", version = "1.19.0"}
 
 aspectj-weaver = {module = "org.aspectj:aspectjweaver", version.ref = "aspectj"}
+
+testcontainernginx = {module = "org.testcontainers:nginx", version.ref = 'nginxContainer'}
```

Comment:
Maybe move up next to the other testcontainers dependency?

Reply:
Done